main/src/test/scala/ParseKey.scala

```diff
@@ -101,7 +116,10 @@ object ParseKey extends Properties("Key parser test") {
       ("Mask: " + mask) |:
       parse(structure, s) {
       case Left(err) => false
-      case Right(sk) => Project.equal(sk, expected, mask)
+      case Right(sk) =>
+        // println(s"${sk}.key == ${expected}.key: ${sk.key == expected.key}")
+        // println(s"${sk.scope} == ${expected.scope}: ${Scope.equal(sk.scope, expected.scope, mask)}")
```

Comment:
remove

Reply:
Actually the right thing to do is convert it into ScalaCheck label so it'll be displayed when it fails:

```scala
        (s"${sk}.key == ${expected}.key: ${sk.key == expected.key}") |:
          (s"${sk.scope} == ${expected.scope}: ${Scope.equal(sk.scope, expected.scope, mask)}") |:
          Project.equal(sk, expected, mask)
```